src/Link.spec.tsx

```diff
@@ -7,7 +7,6 @@ import * as router from 'next/router'
 
 describe('Link', () => {
   it('renders a Link component', async () => {
-    const url = 'example_url'
```

Comment:
Again, this will have conflicts with my changes. I removed most of the test as it wasn't actually using the `mockRouter` or `Link` at all.

I've written a test in `DocsWrapper.spec.tsx` that does use a mocked router, so if we want to update this down the line I'd use that as a reference.